code/obj/item/plants.dm

```diff
@@ -10,6 +10,7 @@ ABSTRACT_TYPE(/obj/item/plant)
 	desc = "You shouldn't be able to see this item ingame!"
 	icon = 'icons/obj/hydroponics/items_hydroponics.dmi'
 	var/brew_result = null // what will it make if it's brewable?
+	var/brew_amount = 20 // how much brew could a brewstill brew if a brewstill still brewed brew?
```

Comment:
I'm not sure `brew_amount` is necessary? What is the benefit of having multiple ways of defining things.
```
brew_result = "water"
brew_amount = 5
```
```
brew_result = list("water"=5)
```

Reply:
Primarily, it's so that older stuff uniformly uses the newer system and I don't have to manually rewrite every single brew_result into a list, since the PR was already pushing itself a bit out of scope. But I'll just do that then.